pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -134,6 +137,12 @@ type MonSpec struct {
 	AllowMultiplePerNode bool `json:"allowMultiplePerNode"`
 }
 
+type MgrSpec struct {
+	Count                int  `json:"count"`
+	PreferredCount       int  `json:"preferredCount"`
+	AllowMultiplePerNode bool `json:"allowMultiplePerNode"`
```

Comment:
I wonder if we could just not add this property. It isn't as critical for the mgr as the mons to be running on separate nodes, but the operator should certainly attempt to schedule them on different nodes if possible.

Reply:
The mgrs will still need to be on separate nodes for host networking I believe. Though I'm still interested in removing this property. I would rather set an anti-affinity on the mgr pods when host networking is enabled so Kubernetes won't schedule them together. 

I don't know what the behavior would be if there is only a single node the mgr could be scheduled on however.